news/14703-condabin

```diff
@@ -0,0 +1,19 @@
+### Enhancements
+
+* Provide `conda init --condabin`, which only adds the `condabin/` directory to the PATH environment variable instead of installing a shell function. (#14703)
```

Comment:
```suggestion
* Provide `conda init --condabin`, which only adds the `$CONDA_PREFIX/condabin` directory to the PATH environment variable instead of installing a shell function. (#14703)
```